docs/source/development/start/setup.rst

```diff
@@ -0,0 +1,165 @@
+.. _contribution_setup_and_basics:
+
+Setup and Basics
+================
+
+.. include:: ../../snippets/slack_support.rstinc
+
+.. _contribution_install:
+
+Installation
+------------
+
+Cloning Repo
+************
+
+First `fork <https://docs.github.com/en/get-started/quickstart/fork-a-repo>`_ the ``lineapy`` repo.
+Then, `clone <https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository>`_
+the forked repo to your development environment:
+
+.. code:: bash
+
+    cd <PATH-TO-DESIRED-LOCATION>
+    git clone <URL-TO-FORKED-REPO>
+
+To keep the forked repo in sync with the original one, set an "upstream":
+
+.. code:: bash
+
+    git remote add upstream https://github.com/LineaLabs/lineapy.git
+
+Setting up Virtual Environment
+******************************
+
+When done cloning, move into the repo and initiate a virtual environment:
```

Comment:
Maybe have a line here saying if you know how to set up your own env (e.g. through conda) to skip this section. I for one have a totally different virtualenv setup.

Reply:
Good point, done.